app/assets/javascripts/app/views/photos_view.js

```diff
@@ -1,26 +1,28 @@
 // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
 
 app.views.Photos = app.views.InfScroll.extend({
+  className: "clearfix",
+
   initialize : function() {
     this.stream = this.model;
     this.collection = this.stream.items;
 
     // viable for extraction
     this.stream.fetch();
-
-    this.setupLightbox();
     this.setupInfiniteScroll();
   },
 
-  postClass : app.views.Photo,
+  postRenderTemplate: function(){
+    this.setupGallery();
```

Comment:
I think instead of calling just that one function you can move the code from `setupGallery` here.